bangazon/bangazon_ui/templates/bangazon_ui/product_list.html

```diff
@@ -3,7 +3,7 @@
 
 {% if product_list %}
     <ul>
-    {% for product in product_list %}
+    {% for paym in product_list %}
```

Comment:
i'll check it . i must've started in the wrong file and then switched over to the correct one for payment in payment_types